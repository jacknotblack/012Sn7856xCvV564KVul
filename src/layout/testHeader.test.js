import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import TestHeader from "./testHeader";

describe("A suite", () => {
  let wrapper;
  const props = {
    children: <div>child</div>
  };

  beforeEach(() => {
    wrapper = shallow(<TestHeader {...props} />);
  });
  it("renders without crashing", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders children", () => {
    expect(
      wrapper
        .find("div>div")
        .at(1)
        .text()
    ).toEqual("child");
  });
});
