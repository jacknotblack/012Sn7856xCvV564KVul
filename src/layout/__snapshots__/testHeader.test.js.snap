// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite should match snapshot 1`] = `
<div>
  <div>
    THIS IS HEADER
  </div>
  <div>
    child
  </div>
</div>
`;
